lib/tasks/seek_upgrades.rake

```diff
@@ -235,6 +236,40 @@ namespace :seek do
     end
   end
 
+  task(add_policies_to_existing_sample_types: [:environment]) do
+    puts '... Adding policies to existing sample types'
+    counter = 0
+    disable_authorization_checks do
+      SampleType.includes(:projects, :assays, :studies).where(policy_id: nil).each do |st|
+        if st.is_isa_json_compliant?
+          st.update_column(:policy_id, st.assays.first.policy_id) if st.assays.any?
+          st.update_column(:policy_id, st.studies.first.policy_id) if st.studies.any?
+        else
+          project = st.projects.first
+          if project.use_default_policy
+            st.update_column(:policy_id, project.default_policy_id)
```

Comment:
this will create a reference to the default policy, which will then also be changed if the ST sharing permissions are changed. I'm also not sure it's right